09-model-mlr.Rmd

```diff
@@ -646,9 +646,27 @@ In this example, we have arrived at the same model that we identified from backw
 
 Backward elimination begins with the model having the largest number of predictors and eliminates variables one-by-one until we are satisfied that all remaining variables are important to the model.
 Forward selection starts with no variables included in the model, then it adds in variables according to their importance until no other important variables are found.
-
-Backward elimination and forward selection sometimes arrive at different final models.
-If trying both techniques and this happens, it's common to choose the model with the larger adjusted $R^2.$
+Notice that, for both methods, we have always chosen to retain the model with the largest adjusted $R^2$ value, even if the difference is less than half a percent (e.g., 0.2597 versus 0.2598).
+One could argue that the difference between these two models is negligible, as they both explain nearly the same amount of variability in the `interest_rate`.
+These negligible differences are an important aspect to model selection.
+It is highly advised that *before* you begin the model selection process, you decide what a "meaningful" difference in adjusted $R^2$ is for the context of your data.
+Maybe this difference is 1% or maybe it is 5%.
+This "threshold" is what you will then use to decide if one model is "better" than another model.
+Using meaningful thresholds in model selection requires more critical thinking about what the adjusted $R^2$ values mean!
+
+Additionally, you should know that backward elimination and forward selection sometimes arrive at different final models.
+This is because the decision for whether to include a variable or not depends on what other variables are already in the model.
+With forward selection, you start with a model that includes no variables and add variables from there.
+In backward selection, you start with a model that includes all of the variables and remove variables from there.
+A variable's "effect" depends on what other variables are in the model. 
+This is especially important if the variables are correlated with each other.
+Backward selection models will naturally discard the variable(s) that explain similar aspects in the response.
+Whereas forward selection models will only allow for a new variable to be added, if it adds new information.
+For this reason, forward selection is also sensitive to the order that variables are added to the model.
+One variable could be included in the model if it was added first, but not included if it was added fifth. 
```

Comment:
I feel like this might confuse matters more than clarify. Ultimately the order in forward selection is deterministic, so I feel like this is more about whether a particular variable would be added to a model if evaluating different sets of variables as opposed to a specific set of variables. Since in this chapter we're talking about using a specific set of variables, I think this is out of scope (or needs more explanation). Therefore I'm tempted to omit these four sentences instead of expanding on them more (as I don't think expanding more without an example is super helpful, but I also don't want to spend too much space on stepwise selection). Thoughts @atheobold?